File: core/src/net/bossdragon/enums/RenderLayers.java
package net.bossdragon.enums;

import net.bossdragon.component.render.Renderable;

/**
 * Higher value is in front, lower is behind.

 * @see Renderable#layer
 */
public interface RenderLayers {
    int WORLD = 0;
    int ENTITIES = 1;
    int HUD = 2;
}

File: core/src/net/bossdragon/enums/RenderLayers.kt
package net.bossdragon.enums

import net.bossdragon.component.render.Renderable

/**
 * Higher value is in front, lower is behind.

 * @see Renderable.layer
 */
object RenderLayers {
    val WORLD = 0
    val ENTITIES = 1
    val HUD = 2
}
